rust/typeql_lang.rs

```diff
@@ -60,6 +60,17 @@ macro_rules! typeql_match {
     }}
 }
 
+#[macro_export]
+macro_rules! typeql_insert {
+    ($($thing_variable:expr),* $(,)?) => {{
+        let variables = [$($thing_variable),*].into_iter().collect::<Result<Vec<_>, ErrorMessage>>();
+        match variables {
+            Ok(variables) => Ok(TypeQLInsert::new(variables)),
+            Err(err) => Err(err),
+        }
+    }}
+}
+
```

Comment:
Not that there's not a top-level `typeql_delete` builder, because a `delete` query requires a `match` at the start.